pkg/products/rhsso/reconciler.go

```diff
@@ -62,12 +62,12 @@ type Reconciler struct {
 	logger        *logrus.Entry
 	oauthv1Client oauthClient.OauthV1Interface
 	KeycloakHost  string
-	ApiUrl        string
+	apiURL        string
```

Comment:
I'm a little suspicious of this field name changing from an uppercase `A` to a lowercase `a`. :thinking: I guess the golint warning for this was telling us to change `Url` to `URL`, right?

Reply:
I believe there was some more problems down the line when it was changed to `ApiURL` so it had to be reverted back to `apiURL` i'll do some more testing around this and see if it has to stay this way for definite :ok_hand: